kymatio/scattering2d/filter_bank.py

```diff
@@ -49,8 +49,6 @@ def filter_bank(M, N, J, L=8):
             for res in range(min(j + 1, max(J - 1, 1))):
                 psi_signal_fourier_res = periodize_filter_fft(
                     psi_signal_fourier, res)
-                # add a trailing singleton dimension to mark it as non-complex
-                psi_signal_fourier_res = psi_signal_fourier_res[..., np.newaxis]
```

Comment:
why is this gone now?

Reply:
it's done here: https://github.com/edouardoyallon/kymatio/blob/kymatio-dev-frontend/kymatio/scattering2d/frontend/torch_frontend.py#L132 this is cleaner! (AFAIR, it was xploiting the fact that the wavelet is real and added a 2nd dimension for torch ; now it just does a potentially complex numpy tensor)

Reply:
Yeah, I saw that later. Good point